src/run.test.ts

```diff
@@ -25,6 +25,7 @@ describe('Set context', () => {
             .spyOn(core, 'getInput')
             .mockImplementation((inputName, options) => {
                if (inputName == 'resource-group') return resourceGroup
+               if (inputName == 'cluster-name') return ''
```

Comment:
This return for cluster name is set for blank string to avoid test failures ?

Reply:
this is a test for how it handles a missing cluster name, which getInput returns as blank

Reply:
it's expected to fail, i just added an explicit case for the value we expect to be blank so that it only returns those two values instead of potentially also returning blank for all unspecified values